elyra/metadata/schemas/code-snippet.json

```
@@ -23,11 +23,18 @@
         },
         "language": {
           "description": "Code snippet implementation language",
-          "type": "string"
+          "type": "string",
+          "ui_hints": {
+            "field_type": "dropdown",
+            "defaultChoices": ["Python", "Java", "R", "Scala", "Markdown"]
```

I'd prefer we use consistent formats.  Currently, all schema fields are lower-case with underscores.
```suggestion
            "default_choices": ["Python", "Java", "R", "Scala", "Markdown"]
```

---

fixed